manifest_staging/charts/secrets-store-csi-driver/README.md

```diff
@@ -34,6 +34,7 @@ The following table lists the configurable parameters of the csi-secrets-store-p
 | `linux.image.repository`                | Linux image repository                                                                                                                                                         | `registry.k8s.io/csi-secrets-store/driver`              |
 | `linux.image.pullPolicy`                | Linux image pull policy                                                                                                                                                        | `IfNotPresent`                                          |
 | `linux.image.tag`                       | Linux image tag                                                                                                                                                                | `v1.3.4`                                                |
+| `linux.image.digest`                       | Image pull from digest instead of tag if specified                                                                                                                                                                 | `sha256:xxxxxx`                                                |
```

Comment:
```suggestion
| `linux.image.digest`                       | Image pull using digest instead of tag if specified (`sha256:xxxxxxx`)                                                                                                                                                                 | ``                                                |
```

The default value is empty!